pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -57,18 +57,43 @@ class CBuilder implements Builder {
   @visibleForTesting
   final Uri? installName;
 
+  /// Definitions of preprocessor macros.
+  ///
+  /// When the value is `null`, the macro is defined without a value.
+  final Map<String, String?> defines;
+
+  /// Whether to define a macro for the current [BuildMode].
+  ///
+  /// The macro name is the uppercase name of the build mode and does not have a
+  /// value.
+  ///
+  /// Defaults to `true`.
+  final bool buildModeMacro;
+
+  /// Whether to define the standard `NDEBUG` macro when _not_ building with
```

Comment:
Is this common practise? Maybe add a link to some documentation where this is commonly used.

Reply:
It disables the `assert` macro in `assert.h`.